lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -402,9 +423,6 @@ def test_http_exception(
         """Check that an exception in calling Elasticsearch is reported
         correctly.
         """
-        if not self.elastic_endpoint:
-            pytest.skip("skipping " + self.test_http_exception.__name__)
-
```

Comment:
Given the appearance in other contexts, I'm puzzled why you saw this as redundant in this test. What did I miss?

Reply:
I approached all this by just removing all the "skips" to see what failed, then added back skips for invocations which failed due to the test not in the right environment.

Nothing needed fix-ups for this one or the one below.

Reply:
Hmm. Odd, since there's a query below that uses `self.elastic_endpoint` and it's just a bit disconcerting that this apparently doesn't cause a problem ...

Reply:
Well, @dbutenhof, I guess if `self.elastic_endpoint` has no value then it must result in an exception which is just what this test is designed to provoke...no?  🤣